llvm/test/Transforms/InstCombine/freeze.ll

```diff
@@ -1138,15 +1138,26 @@ define i32 @propagate_drop_flags_or(i32 %arg) {
   ret i32 %v1.fr
 }
 
+define i32 @propagate_drop_flags_trunc(i64 %arg) {
+; CHECK-LABEL: @propagate_drop_flags_trunc(
+; CHECK-NEXT:    [[ARG_FR:%.*]] = freeze i64 [[ARG:%.*]]
+; CHECK-NEXT:    [[V1:%.*]] = trunc i64 [[ARG_FR]] to i32
+; CHECK-NEXT:    ret i32 [[V1]]
+;
+  %v1 = trunc nsw nuw i64 %arg to i32
+  %v1.fr = freeze i32 %v1
+  ret i32 %v1.fr
+}
+
 !0 = !{}
 !1 = !{i64 4}
 !2 = !{i32 0, i32 100}
 ;.
 ; CHECK: attributes #[[ATTR0:[0-9]+]] = { nocallback nofree nosync nounwind speculatable willreturn memory(none) }
 ; CHECK: attributes #[[ATTR1]] = { nounwind }
 ;.
-; CHECK: [[META0:![0-9]+]] = !{}
-; CHECK: [[META1:![0-9]+]] = !{i64 4}
```

Comment:
Seems unrelated.

Reply:
Maybe rebase ontop of NFC updating this test?

Reply:
@goldsteinn  Should I rebase to the last commit in the trunk, and regenerate the tests?

Reply:
yes

Reply:
I rebased on https://github.com/llvm/llvm-project/commit/3522de9e412fbaa6d7344cc31bd43d1be6c95d59 but nothing new.

Reply:
> I rebased on https://github.com/llvm/llvm-project/commit/3522de9e412fbaa6d7344cc31bd43d1be6c95d59 but nothing new.

BTW, It was the latest commit when I rebased.

Reply:
> @goldsteinn Should I rebase to the last commit in the trunk, and regenerate the tests?

I think what goldsteinn said is to regenerate this test with the trunk and open a separate NFC PR.

Reply:
@dtcxzyw, Sorry. BTW, why should we do this if it's going to be the same result?
I mean it's just the names that is chganged, it may be happen if we regenerate tests of some other files.

Reply:
It is fine to just keep it as is. You can just drop these changes manually.
But we will encounter the same problem next time we update this file :(

Reply:
Will we have problems if we keep it as is? If not we can keep it.

Reply:
And isn't it ready to land? 😞